crates/proof-of-sql/src/proof_primitive/dory/dynamic_build_vmv_state.rs

```diff
@@ -1,7 +1,8 @@
 use super::{
-    dynamic_dory_helper::{compute_dynamic_v_vec, compute_dynamic_vecs},
-    DeferredGT, G1Affine, VMVProverState, VMVVerifierState, F,
+    dynamic_dory_helper::compute_dynamic_v_vec, DeferredGT, DoryScalar, G1Affine, VMVProverState,
+    VMVVerifierState, F,
 };
+use crate::proof_primitive::dynamic_dory_and_hyrax_common_utils::standard_basis_helper::compute_dynamic_vecs;
```

Comment:
NIT: Since we may have additional commitment schemes in the future, which might also end up sharing this code, it might be better to rename the utils module something more abstract like

```
commitment_scalar_common_utils
```
or
```
scalar_common_utils
```
or something else that makes sense to you.

Reply:
"dynamic" and "matrix" are the words that come to my mine